### PR TITLE
15 implement all swap transactions available in lssvmrouter

### PR DESCRIPTION
**Pull request by bisbist**

### CODE REVIEW

File: webui/src/components/Swap/index.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
+import React from 'react'
+import config from '../../config'
+import { provider, contracts } from '../../environment'
+import SwapNFTsForToken from './SwapNFTsForToken'
+import SwapETHForAnyNFTs from './SwapETHForAnyNFTs'
+import SwapERC20ForAnyNFTs from './SwapERC20ForAnyNFTs'
```
Comment (bbist) [inline webui/src/components/Swap/index.js:6]: Where's this file? I think you forgot to commit it, or comment out this line.

File: webui/src/components/Swap/index.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+import RobustSwapETHForAnyNFTs from './RobustSwapETHForAnyNFTs'
+import RobustSwapETHForSpecificNFTs from './RobustSwapETHForSpecificNFTs'
+import RobustSwapERC20ForAnyNFTs from './RobustSwapERC20ForAnyNFTs'
```
Comment (bbist) [inline webui/src/components/Swap/index.js:9]: File not committed.

File: webui/src/components/Swap/index.js [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+import SwapERC20ForSpecificNFTs from './SwapERC20ForSpecificNFTs'
```
Comment (bbist) [inline webui/src/components/Swap/index.js:10]: File not committed.

File: webui/src/components/Swap/index.js [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+                            name: routerName,
+                            createContract: routers[routerName].createContract,
+                        }} />
+                    ) : swapType == Types.RobustSwapETHForSpecificNFTs ? (
+                        <RobustSwapETHForSpecificNFTs router={{
+                            name: routerName,
+                            createContract: routers[routerName].createContract,
+                        }} />
+                    ) : swapType == Types.RobustSwapERC20ForAnyNFTs ? (
+                        <RobustSwapERC20ForAnyNFTs router={{
```
Comment (bbist) [inline webui/src/components/Swap/index.js:191]: Components not committed. Either commit them or comment out these.

File: webui/src/pages/Home.js [hunk 5]
```diff
@@ -11,6 +12,7 @@ const Home = () => {
       <div style={{ display: 'flex', flexDirection: 'row', marginTop: 20 }}>
         <PoolCreator />
         <TokenSwapper selectedPair={pair} />
```
Comment (bbist) [inline webui/src/pages/Home.js:14]: Now that Swap component is partly functional, can TokenSwapper be removed?

**Review by bbist — CHANGES_REQUESTED** [review]

File: webui/src/pages/Home.js (hunk 5 above)
Comment (bisbist) [reply webui/src/pages/Home.js:14]: TokenSwapper can be removed now, I'll do it right now.

**Review by bbist — APPROVED** [review]

**Pull request merged**